### PR TITLE
#166151259 reset cache on muscle delete

### PR DESCRIPTION
**Pull request by jmusila**
#### What does this PR do?
- Clear cache on deleting a muscle

#### Description of the Tasks to be completed?

- [x] reset cache on deleting 

#### How should this be manually tested?
Clone the repository to your local branch

$` git clone https://github.com/andela/wg-baqi.git`

$ `cd to the wg-baqi directory, create a virtual environment and install the dependencies`

$ `cd wg-baqi`

$` virtualenv env`

$ `source env/bin/activate`

$ `pip install -r requirements.txt`

`Activate bootstrap and run the server`

$` invoke bootstrap-wger --settings-path ./wger/settings.py --no-start-server`

$ `python manage.py runserver`

- Login as admin and visit the page `http://127.0.0.1:8000/en/exercise/muscle/admin-overview/`
- Go to muscle menu and add a muscle 
- Go back to the muscle overview and confirm that the must has been added
- Go back to the muscles menu and delete the muscle you added and verify that the muscle no-longer exists on the muscle overview.

#### What are the relevant pivotal tracker stories?
#166151259

### CODE REVIEW

File: wger/exercises/views/muscles.py [hunk 1]
```diff
@@ -61,7 +65,7 @@ def get_context_data(self, **kwargs):
 
 
 class MuscleAdminListView(LoginRequiredMixin,
-                          PermissionRequiredMixin, MuscleListView):
+                          PermissionRequiredMixin, ListView):
```
Comment (ceexon) [inline wger/exercises/views/muscles.py:67]: Why did you switch from MuscleListView which seems to have been defined by the original developers to ListView?

File: wger/exercises/models.py [hunk 2]
```diff
@@ -20,6 +20,7 @@
 import logging
 import bleach
 
+# from django.db.models import Q
```
Comment (wangonya) [inline wger/exercises/models.py]: Kindly remove the commented code here.

**Review by wangonya — CHANGES_REQUESTED** [review]

**Comment by wangonya** [comment]
Tested locally. Working 👍🏾

File: wger/exercises/views/muscles.py (hunk 1 above)
Comment (jmusila) [reply wger/exercises/views/muscles.py:67]: I overrode the `get_queryset` so that's why I added ListView to allow return the list of all muscles

**Review by willeswa — APPROVED** [review]
LGTM

**Review by wekesa931 — COMMENTED** [review]
Good implementation. On your PR description, you could point out the advantage of the implementation and compare it to the original implementation.

**Pull request merged**